### ISSUE TITLE
Screenreader compatibility with reading headings

### ISSUE THREAD

**Issue by brittweinstein** [Connexions/webview#1756]
@brittweinstein commented on [Thu Jul 26 2018](https://github.com/Connexions/content-managers/issues/166)

**SCENARIO** _Briefly describe the issue._

Make sure that screenreaders read all headings and sub-headings in readings -- needs investigation. In V-sprint planning we discussed this ticket cannot be handled by a Tutor fix alone, so we probably want to take this one out of the Tutor category and place elsewhere. I am proposing that we put this on book tools because Kathi seems to think the fix requires us to modify the transforms, but I am open to discussion if that doesn't make sense. 

----------------

**Frequency:** _How many times were you able to reproduce this issue?_

----------------
**Steps to Reproduce the issue:** _Who, Where and How?_

- **Product**: [x] Tutor  [ ] Accounts  [ ] Exercises  [ ] BigLearn  [ ] CNX/Webview
- **User Type**: [ ] Teacher  [ ] Student  [ ] Admin  [ ] Content/QA
- **Username**: 
- **URL, Course number or Course type**: 

- **Steps**: 

  1. 

----------------
**Environment:**

- **Instance**: [ ] production  [ ] qa  [ ] staging  [ ] dev  [ ] content
- **OS**: [ ] Mac OSX  [ ] Windows  [ ] iOS  [ ] Android
- **Browser**: [ ] Chrome  [ ] Firefox  [ ] Safari  [ ] Opera  [ ] IE  [ ] Edge
- **Browser Version**: 

----------------
**Impact on a User:**

----------------
**Suggested Urgency:** _Priority to fix?_
[ ] Low  [ ] Medium  [ ] High  [ ] Immediate (_showstopper_)

----------------
**Images or Screenshots** (_attach below_)
![Screen_Shot_2017-11-15_at_10.04.47_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/e57ef6f3-e031-4b52-8d40-33a16b5ea1a0)![Screen_Shot_2017-11-15_at_10.04.23_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/10bd31d4-9455-4f9d-89ff-a06a50deb0c3)![Screen_Shot_2017-11-15_at_10.04.01_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/2032274c-5616-409f-9b5a-97a326c9f993)![Screen_Shot_2017-11-15_at_10.03.30_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/8eb61dfb-e001-4a1e-84d2-2244c966d030)![Screen_Shot_2017-11-15_at_10.03.14_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/a7b8896e-324c-4292-a9dc-c9374716e08c)![Screen_Shot_2017-11-15_at_10.04.47_AM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/c7e64570-2b6a-428d-9194-08d4c2248c0f)

More screenshots: https://trello.com/c/lNh4DSLT/536-screenreader-compatibility-with-reading-headings

**Comment by openstaxalina** [comment]
I need to create a new issue that has the relevant information for CNX.  Closing this issue.
